.isort.cfg

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+[settings]
```

Comment:
I separated the config to use a preferred format. If it is better to use old `setup.cfg`, I can add this section back to `setup.cfg`

Reply:
I think to keep the amount of files as minimal as possible in the root repo it would be nice to keep this within the `setup.cfg` file.

Reply:
Done